recipes/hdmedians/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "hdmedians" %}
+{% set version = "0.13" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 230f80e064d905c49a1941af1b7e806e2f22b3c9a90ad5c21fd17d72636ea277
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython >=0.23
+    - numpy
+    - pip
+    - python
+    - nose
+  run:
+    - numpy
+    - python
+test:
+  imports:
+    - hdmedians
+
+about:
+  home: http://github.com/daleroberts/hdmedians
+  license: GPL3
```

Comment:
There is a conflict between the license named in the `setup.py` (GPL) and in the actual license file in the repo (Apache-2.0). Let's figure out which is correct and include the `license_file` in the recipe.

Reply:
I believe the difference stems from the fact that there hasn't been a new release since the license changed. See [this issue](https://github.com/daleroberts/hdmedians/issues/1) where the maintainer kindly agreed to change the license. I've submitted [a PR](https://github.com/daleroberts/hdmedians/pull/3/files) to fix the setup.py.

Reply:
Ok, but you still need to distribute the Apache license

Reply:
Thanks, for now I've left GPL3 and included the corresponding license file. I'll update this to Apache once there's a new release of hdmedians.

Reply:
sounds good!